test/integration/framework/etcd.go

```diff
@@ -136,6 +138,27 @@ func startEtcd() (func(), error) {
 	if err := cmd.Start(); err != nil {
 		return nil, fmt.Errorf("failed to run etcd: %v", err)
 	}
+
+	i := 0
```

Comment:
Prefer `var i int` when its initial value is the default

https://dave.cheney.net/practical-go/presentations/qcon-china.html#_use_a_consistent_declaration_style

Reply:
Done. Thanks.